yarn.lock

```diff
@@ -4551,7 +4551,7 @@ etag@~1.8.1:
   resolved "https://registry.yarnpkg.com/etag/-/etag-1.8.1.tgz#41ae2eeb65efa62268aebfea83ac7d79299b0887"
```

Comment:
The `yarn.lock` file change should also be in the next commit with the `package.json` change.